src/hotspot/cpu/zero/zeroInterpreter_zero.cpp

```diff
@@ -138,7 +138,9 @@ intptr_t narrow(BasicType type, intptr_t result) {
     case T_DOUBLE:
     case T_VOID:
       return result;
-    default  : ShouldNotReachHere();
+    default:
+      ShouldNotReachHere();
+      return result;
```

Comment:
Perhaps add a `// silence compiler` comment to the return?

Reply:
Right. Added in new revision.